eq-author/src/tests/setup/throwOnConsole.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-console, import/unambiguous */
 const actualConsoleError = console.error.bind(console);
 
-console.error = jest.fn(function(msg) {
-  actualConsoleError(msg);
+console.error = jest.fn(function(msg, ...rest) {
+  actualConsoleError(msg, rest);
```

Comment:
Should this spread as well?
```suggestion
  actualConsoleError(msg, ...rest);
```

Reply:
Yes.  I think it's better to spread here as well.